public/app/features/browse-dashboards/RecentlyDeletedPage.tsx

```diff
@@ -53,18 +53,26 @@ const RecentlyDeletedPage = memo(() => {
       <Page.Contents>
         {searchState.result && (
           <>
-            <ActionRow
-              state={searchState}
-              getTagOptions={stateManager.getTagOptions}
-              getSortOptions={getGrafanaSearcher().getSortOptions}
-              sortPlaceholder={getGrafanaSearcher().sortPlaceholder}
-              onLayoutChange={stateManager.onLayoutChange}
-              onSortChange={stateManager.onSortChange}
-              onTagFilterChange={stateManager.onTagFilterChange}
-              onDatasourceChange={stateManager.onDatasourceChange}
-              onPanelTypeChange={stateManager.onPanelTypeChange}
-              onSetIncludePanels={stateManager.onSetIncludePanels}
-            />
+            <Stack direction="column">
+              <FilterInput
+                placeholder={t('recentlyDeleted.filter.placeholder', 'Search for dashboards')}
+                value={searchState.query}
+                escapeRegex={false}
+                onChange={(e) => stateManager.onQueryChange(e)}
```

Comment:
```suggestion
                onChange={stateManager.onQueryChange}
```
Nit: I think this can be simplified.